Create Meeting 121.md (#386)
ethereum/pm: All Core Devs Meetings/Meeting 121.md

um I think Pooja you had put a couple EIPs on the agenda uh EIP 2364 and 2464 which are basically the eth 64 and eth 65 protocol EIPs uh as I understand that the issue is like both of those are shipped but the EIPs are still like in draft right

**Pooja**

right so the main issue here is uh EIP-2481 which is for Eth 66. that is uh in the last call actually the last call duration has also passed and we would want to move that to the final status

but the problem is that proposal requires Eth 65 which is 2464 and Eth 65 requires Eth 64 which is two three six four and uh both these proposals are still in draft status so we would want that these two

proposals should move to the final before we could make move EIP 2481 to the final status I'm happy to make the request to request a status change it's just that we wanted to make sure that it

is in knowledge of geth team if anyone from get team wants to volunteer and do that fair enough and if not then we can uh do that and we would need just author's approval

> **??**

Reviewer: ```suggestion
**Felix**
```

so thanks for the initiative what I can say is that I think the last time we tried
to do something like this there was like a huge amount of backlash for some reason because then people came on and you know like wanted to actually see some justification for these EIPs even

though they are like four years old or something I don't really know like this is definitely something that I would like to avoid so I feel like these EIPs they have been you know we don't even

> use 864 anymore it's already like past you know it's it's it's basically it's already happened like it we there is really no reason not to move it to the final because

Reviewer: ```suggestion
use Eth 64 anymore it's already like past you know it's basically it's already happened like it we there is really no reason not to move it to the final because
```

it's not even supported anymore and I mean the mechanism in it is obviously still supported because it's carried over into the newer protocol versions but the protocol version itself has

already advanced like beyond it's like already like after it's end of life now so I feel like it's like from this point of view moving these eaves to the final is like

totally justified and I really just want to avoid getting the same kind of weird discussions again that we had last time where someone then wanted to like you know for four reasons yeah 

**Tim**

yeah yeah so let's just try and like if I think peter is the author on both so he'd have to actually accept the the change if I understand but like yes

**Pooja**

right yeah yeah I'll make a pull request to move the status from draft to review that would be the first step and

> **??**

Reviewer: ```suggestion
**Felix**
```

guys can you just commit it to the final I mean doesn't it no no no is it really necessary

**Pooja**

okay we have some EIP editors on the call if they have you I just have pasted um like a link to the earlier pull request

which we created for EIP 2481 where we received some comments from EIP editors mentioning that these two proposals should be moved so if we can directly make it to the final happy to do that

**Micah**

I can answer that quickly um I'm not I'm not a fan of skipping straight to final because it encourages people to drag their feet because if you drag your feet long enough eventually you can just avoid the bureaucracy and as much as I hate bureaucracy um I don't want to

create perverse incentives for people to uh just like not to go through the process knowing that if they wait long enough they eventually they can avoid it um

> **??**

Reviewer: ```suggestion
**Felix**
```

in review so can we please can we try to avoid putting it in the review I mean it's okay if they move to the last call or something but they like

the review seriously this is it's inappropriate because what is possibly gonna happen we just put it to the last call and then maybe even wait if you if you if we have to I don't know it's a special case but

**Micah**

let's talk about this in discord uh only because I suspect 90 of people remain in this call probably don't care yeah okay

# Announcement

**Tim**

cool um and okay so last thing and that was it in terms of content but uh next friday support for 1559 um yeah so if you are kind of an application or wallet or just generally interested in kind of broad adoption for eip-1559 you can join that uh we'll post the link

in all core dev and there's an issue on the ethereum pm repo um for it and yeah that's pretty much all we have um thanks everybody I appreciate uh you staying real

> **??**

Reviewer: ```suggestion
**Trent**
```
